provider/go.mod

```diff
@@ -1,12 +1,242 @@
 module github.com/pulumi/pulumi-venafi/provider
 
-go 1.16
-
-replace github.com/Venafi/terraform-provider-venafi => github.com/pulumi/terraform-provider-venafi v0.10.2-0.20220421210328-b69b527db299
+go 1.19
```

Comment:
Just wondering – should we be bumping the go version where applicable for all provider updates?

Reply:
I think so, especially when the go version is old enough that it lacks indirect dependencies and/or go.sum is missing.